multiapps-controller-web/src/main/java/org/cloudfoundry/multiapps/controller/web/configuration/DatabaseConfiguration.java

```diff
@@ -96,11 +94,4 @@ private SpringLiquibase getLiquibaseTemplate(DataSource dataSource, String chang
         return springLiquibase;
     }
 
-    @Inject
-    @Bean
-    @DependsOn("persistenceChangelog")
-    public SpringLiquibase coreChangelog(DataSource dataSource) {
```

Comment:
Why is the core changelog deleted? Was it used somewhere?

Reply:
Because the Liquibase changes from the core module were moved to the persistence module. Now they're all in one place. Finally! \o/

Also, see the bean above this one. There's still a SpringLiquibase bean.